### PR TITLE
Calculating Max Cigarattes to display dynamically

### LINKED ISSUE #440: When there are too many cigarettes, show 2 full lines
Right now, when the number of cigarettes is >50, we limit to show only 50 cigarettes:
https://github.com/amaurymartiny/shoot-i-smoke/blob/f1c67a4cf04d87d9bc82b3414453f582e8b18c46/App/components/Cigarettes/Cigarettes.tsx#L54-L55

However, it's 40 cigarettes and 180 cigarettes seem to have the same number of cigarettes:
Screenshot_20200205-183713 Screenshot_20200205-183718

The small challenge is we don't know how many cigarettes to show to fill 2 lines, it depends on the phone's screen size. On my small phone, 50 almost fills 2 lines. On the above one, it seems like we need ~64.

What we can probably do is 
- either put a max of ~80 cigarettes, and hide the 3rd line
- alternatively, dynamically calculate a cigarette's width, the screen width, and do a division

### PR DESCRIPTION
**Pull request by sarthakpranesh**
Calculating the total number of cigarettes it will take to completely fill two rows dynamically according to the screen width and assigning it to `showMaxCigarettes`

closes #440 

Signed-off-by: sarthakpranesh <sarthak.pranesh2018@vitstudent.ac.in>

### CODE REVIEW

**Comment by amaurymartiny** [comment]
Wow, thanks, I'll have a look tonight.

File: App/Screens/Home/Home.tsx [hunk 1]
```diff
@@ -63,6 +63,32 @@ function getCigarettesMargin(count: number): number {
...
+		count <= THRESHOLD.FIRST
+			? SIZES.BIG
+			: count <= THRESHOLD.SECOND
+			? SIZES.MEDIUM
+			: count <= THRESHOLD.THIRD
+			? SIZES.BIG
+			: count <= THRESHOLD.FOURTH
+			? SIZES.MEDIUM
+			: SIZES.SMALL
+	);
```
Comment (amaurymartiny) [inline App/Screens/Home/Home.tsx:81]: This code is used twice, could you factor it out in a separate function `getCigarettesHeight`?

**Review by amaurymartiny — COMMENTED** [review]
LGTM 👍 I have just one small refactor nit

File: App/Screens/Home/Home.tsx (hunk 1 above)
Comment (sarthakpranesh) [reply App/Screens/Home/Home.tsx]: Changes applied :+1:

**Review by amaurymartiny — APPROVED** [review]
thanks!

**Pull request merged**